main/src/ca/uwaterloo/flix/util/ParOps.scala

```diff
@@ -52,6 +52,18 @@ object ParOps {
       case (k, v) => (k, f(v))
     }.toMap
 
+  /**
+    * Apply the given fallible function `f` to each value in the map `m` in parallel,
+    * returning the resulting map if all calls are successful.
+    */
+  @inline
+  def mapValuesFallible[K, A, B, E](m: Map[K, A])(f: A => Validation[B, E])(implicit flix: Flix): Validation[Map[K, B], E] = {
```

Comment:
rename to `parMapSeq`

Reply:
Wouldn't that make it sound like it corresponded to `parMap` instead of `mapValues`?

Reply:
Maybe parMapValuesSeq

Reply:
Should `mapValues` then also be renamed to `parMapValues`?

Reply:
Ah, maybe without the par

Reply:
Actually, looking at the other functions, it looks like they should all just be called `par`-, right?